dsc/cli.py

```diff
@@ -155,7 +214,7 @@ def finalize(ctx: click.Context, email_recipients: str) -> None:
     is_flag=True,
     help=(
         "Display the operations that would be performed using the "
-        "specified command without actually running them"
+        "sync command without actually running them"
     ),
 )
 def sync(
```

Comment:
Commenting here on `sync` command: I might propose moving this below/after the `create_batch` command but above/before `submit`.  Basically, have the CLI commands be in the expected order of invoking for a happy path.

Reply:
I like this idea as well